Makefile

```diff
@@ -210,21 +210,17 @@ terraform/format: | guard/program/terraform
 	@ echo "[$@]: Successfully formatted terraform files!"
 
 hcl/%: FIND_HCL := find . $(FIND_EXCLUDES) -type f \( -name "*.hcl" \)
-hcl/validate: | guard/program/terraform
```

Comment:
Had this target to avoid blanking out improperly formatted files when running the format target directly. Its been a minute so not sure if this is a concern with tf and hcl but it was def a problem with json formatting

Reply:
Added it back